ci/scripts/acceptance-tests

```diff
@@ -41,6 +41,10 @@ bosh -n upload-stemcell $stemcell_xenial_path
 echo "----- Uploading BPM"
 bosh -n upload-release $bpm_release_path
 
+echo "----- Uploading os-conf (used for tests only)"
+bosh -n upload-release --sha1 386293038ae3d00813eaa475b4acf63f8da226ef \
+  https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=22.1.2
+
```

Comment:
Why was this added now? I thought this doesn't change any of the tests? And would it be possible to consume the other releases from bosh.io directly as well?

Reply:
Just a small refactor so that all releases are uploaded in the same place before running the tests. This was previously being added via a URL in https://github.com/cloudfoundry/haproxy-boshrelease/blob/master/acceptance-tests/bosh_helpers.go#L50-L57 

>  And would it be possible to consume the other releases from bosh.io directly as well?
yes you could consume the BPM release directly from bosh.io however since BPM is also included in the final bosh release tarball, doing it via the pipeline makes it easier to track which version we have